package.json

```diff
@@ -50,7 +50,7 @@
             "plugin": "fzf",
             "version": "0.28.0"
         },
-        "zplugin-ices": {
+        "zinit-ices": {
```

Comment:
That's supported, right?

Reply:
Yes, it is :)

Reply:
yes, zplugin was kept for maintaining backward compatibility, but they are interchangeable.